FeedbackSample/Bridging-Header.h

```diff
@@ -0,0 +1,10 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import <ForeSee/ForeSee.h>
+#import <ForeSeeFeedback/ForeSeeFeedback-Swift.h>
```

Comment:
I wonder how this should be done in an objc project?

Reply:
I think this is OK for now, unless a client asks for it. Unless @sleeke has another idea? In any event, I guess it should be another ticket?